changelog/unreleased/issue-3247

```diff
@@ -1,8 +1,8 @@
 Change: Empty files now have size of 0 in restic ls --json output
```

Comment:
Suggest "Change: Empty files now have size of 0 in `ls --json` output" (the restic context is a given).